### PR TITLE
Feat name change

### PR DESCRIPTION
**Pull request by doktordirk**
sortof pending finishing https://github.com/SpoonX/aurelia-orm/pull/66 to detect name mix-ups

### CODE REVIEW

**Comment by doktordirk** [comment]
@RWOverdijk this

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ This library plays nice with the [Sails.js framework](http://sailsjs.org).
 
 > To keep up to date on [Aurelia](http://www.aurelia.io/), please visit and subscribe to [the official blog](http://blog.durandal.io/). If you have questions, we invite you to [join us on Gitter](https://gitter.im/aurelia/discuss). If you would like to have deeper insight into our development process, please install the [ZenHub](https://zenhub.io) Chrome Extension and visit any of our repository's boards. You can get an overview of all Aurelia work by visiting [the framework board](https://github.com/aurelia/framework#boards).
 
+## Important note
+
+This plugin is now registered with npm and jspm as `aurelia-orm`. This simplifies the installation. With this change `aurelia-orm` will reference to [aurelia-api](https://github.com/spoonx/aurelia-api) as `aurelia-api` only (loosing the `spoonx/..` prefix). You will need to update your project's package.json accordingly to prevent forks.
```
Comment (RWOverdijk) [inline README.md]: I'd simplify this. Something like:

> We've simplified installation and usage! This plugin should now be installed using `jspm i aurelia-orm` or (for webpack) `npm i aurelia-orm`. Make sure you update all references to `spoonx/aurelia-orm` and `spoonx/aurelia-api` and remove the `spoonx/` prefix (don't forget your `config.js`, `package.json` and imports).

Comment (doktordirk) [inline README.md]: and bundles

Comment (RWOverdijk) [inline README.md]: Yeah, that too

**Comment by doktordirk** [comment]
@RWOverdijk merged accidentally with the other PR.most insteresting here was the new README

**Pull request merged**